ado/ipacheckreadreplace.ado

```diff
@@ -83,16 +93,31 @@ qui {
 		di as error "selectmultiple specified for `r(N)' corrections entries without oldvalue"
 		error 498
 	}
-	cap assert missing(`oldvalue') if missing(`selectmultiple')
-	if _rc {
-		count if !missing(`oldvalue') & missing(`selectmultiple')
-		di as error "oldvalue specified for `r(N)' corrections entries without selectmultiple"
-		error 498
-	}
 	
-	tempfile selectmult allcorrections
+	***************** remove entries that should be dropped ********************
+	tempfile selectmult allcorrections droplist
 	save `allcorrections'
-
+	if !missing("`drop'") {
+		cap tostring `drop', replace
```

Comment:
Why is this result being captured? Are you just trying to abbreviate the longer, but more precise code which would be:
```
cap confirm string variable `drop'
if _rc {
    tostring `drop', replace
}
```
I am concerned by just capturing the result of tostring directly you may be preventing a more informative error message.